src/braket/pennylane_plugin/ahs_device.py

```diff
@@ -66,12 +67,7 @@ class BraketAhsDevice(QubitDevice):
 
     operations = {"ParametrizedEvolution"}
 
-    def __init__(
-            self,
-            wires: Union[int, Iterable],
-            device: Device, *,
-            shots: int = 100
-    ):
+    def __init__(self, wires: Union[int, Iterable], device: Device, *, shots: int = 100):
```

Comment:
It's because of `black`. My editor runs black whenever I save a file. I'll change the settings so that they are more consistent with AWS's formatting criteria.